cuqi/distribution/_gaussian.py

```diff
@@ -673,6 +673,13 @@ def get_sqrtprec_from_sqrtprec(dim, sqrtprec, sparse_flag):
         logdet = np.sum(-np.log(precision))
         rank = dim
 
+    # sqrtprec is LinearOperator (only for RTOs)
+    elif isinstance(sqrtprec, spa.linalg.LinearOperator):
+        sqrtprec = sqrtprec
+        if hasattr(sqrtprec, 'logdet'):
```

Comment:
If logdet is not defined, what happens?

Reply:
It's really the user's choice/responsibility if they give `logdet` or not. If they give `logdet`, we will record it and then `logpdf` can be computed; if not, `logpdf` can't be computed. 

For large matrices, it's not easy to compute `logdet`, but luckily we don't need it for samplers like `LinearRTO`

Reply:
This is OK for now.  I will create an issue related to the unormalized logpdf

Reply:
There is an old issue related to this. See #16

Reply:
> There is an old issue related to this. See #16

Hahaha, it turned out to be an echo from two years ago : )